src/lightning_habana/pytorch/plugins/fp8_jsons/maxabs_quant.json

```diff
@@ -0,0 +1,18 @@
+{
+    "method": "HOOKS",
+    "mode": "QUANTIZE",
+    "observer": "maxabs",
+    "scale_method": "maxabs_hw",
+    "whitelist": {
+        "types": [],
+        "names": []
+    },
+    "blacklist": {
+        "types": [],
+        "names": [
+            "lm_head"
+        ]
+    },
+    "dump_stats_path": "/tmp/hqt_output/measure",
```

Comment:
this path might not work always

Reply:
Updated
1. Let user pick json path. if none is provided, use HABANA_LOGS from env as default.
2. Create a new json in same path to avoid modifying package jsons.